cpp/common/src/codingstandards/cpp/Emergent.qll

```diff
@@ -6,44 +6,10 @@ import cpp
 module C11 {
   abstract class EmergentLanguageFeature extends Element { }
 
-  class AlignAsAttribute extends EmergentLanguageFeature, Attribute {
```

Comment:
Did you mean to make these changes here?

Reply:
I did mean to, yes. This covers all the amendments that aren't blocked from amoundment3.

However, I reverted the deletion of `_Generic`, and I changed `amendments.csv` to mark the `RULE-1-4` as not yet implemented, so that we don't forget to revisit that once generics are landed.

Reply:
Also reverted deletions of atomics, threads, since those rules haven't been merged yet, and marked `RULE-1-4` from amendment 4 not yet implemented.